map/src/infoblock/components/graph/GpxGraphProvider.jsx

```diff
@@ -91,23 +124,67 @@ const GpxGraphProvider = ({width}) => {
                 }
 
                 let dataTab = {
-                    "Distance": Math.round(sumDist) / 1000,
-                    "Elevation": ele,
-                    "ElevationSRTM": eleSRTM,
-                    "Speed": speed,
+                    [DISTANCE]: Math.round(sumDist) / 1000,
+                    [ELEVATION]: ele,
+                    [ELEVATION_SRTM]: eleSRTM,
+                    [SPEED]: speed,
                 };
                 result.push(dataTab);
             });
             return {res: result, minEle: minEle, maxEle: maxEle, minSpeed: minSpeed, maxSpeed: maxSpeed};
         }
     }, [data]);
 
+    function checkShowData(value) {
+        return value === '' ? false : value;
+    }
 
     return (<>
-            {graphData &&
+            {ctx.currentObjectType === ctx.OBJECT_TYPE_LOCAL_CLIENT_TRACK &&
+                <div style={{marginLeft: '15px', marginTop: '-10px'}}>
```

Comment:
try

style => sx
marginLeft => ml
marginTop => mt

should work good with div